### PR TITLE
Mesos TLS Kerberos Support

### PR DESCRIPTION
**Pull request by cyan-confluent**

### CODE REVIEW

File: debian/base/include/etc/confluent/docker/mesos-setup.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ -z $SKIP_MESOS_AUTO_SETUP ]; then
```
Comment (maxzheng) [inline debian/base/include/etc/confluent/docker/mesos-setup.sh]: How come u need $SKIP.. even though you are already checking MESOS_SANDBOX? As long as the image works when MESOS_SANDBOX is not set, then that seems safe.

File: debian/base/include/etc/confluent/docker/mesos-setup.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        keytool -import \
+                -trustcacerts \
+                -alias root \
+                -file $MESOS_SANDBOX/.ssl/ca-bundle.crt \
+                -storepass changeit \
+                -keystore /tmp/kafka-truststore.jks -noprompt
+
+
+        if [ "$KERBEROS_ENABLED" = "true" ]; then
+            export KAFKAREST_OPTS="-Djava.security.auth.login.config=/tmp/client-jaas.conf -Djava.security.krb5.conf=/tmp/krb5.conf"
```
Comment (maxzheng) [inline debian/base/include/etc/confluent/docker/mesos-setup.sh]: Who is providing /tmp/client-jaas.conf & /tmp/krb5.conf?

File: debian/base/include/etc/confluent/docker/mesos-setup.sh (hunk 1 above)
Comment (cyan-confluent) [reply debian/base/include/etc/confluent/docker/mesos-setup.sh]: yes it's safe for all non-users of mesos. But the idea here is that a small minority of mesos users may still prefer a manual configuration than an out of the box experience. I'm accommodating for that case.

File: debian/base/include/etc/confluent/docker/mesos-setup.sh (hunk 2 above)
Comment (cyan-confluent) [reply debian/base/include/etc/confluent/docker/mesos-setup.sh]: These two files are provided by secret configuration by:
This is on the marrthon/mesos side only, which is why you have to toggle manually in the mesos env:

dcos security secrets create -f /universe-workspace/confluent-aux-universe/config/client-jaas.conf jaas
dcos security secrets create -f /universe-workspace/confluent-aux-universe/config/krb5.conf krb5

**Pull request closed without merging**